### PR TITLE
[devtools-translate] Portuguese - What's New in DevTools (Chrome 112) #5600

### LINKED ISSUE #5600: [devtools-translate] Portuguese - What's New in DevTools (Chrome 112)
Post: https://developer.chrome.com/blog/new-in-devtools-112/
Translated by: Mar-23

Translator: @alvarocamillont
Reviewers: @alvarocamillont, @khaosdoctor

### PR DESCRIPTION
**Pull request by alvarocamillont**
Fixes #5600

Changes proposed in this pull request:

Add portuguese translations for What's New in DevTools M112

### CODE REVIEW

**Comment by alvarocamillont** [comment]
@jecfish @khaosdoctor

File: site/pt/blog/new-in-devtools-112/index.md [hunk 1]
```diff
@@ -26,202 +25,234 @@ draft: true
...
+Se você selecionar um script ignorado, o botão **Configurar** o levará para
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/9gzXiTYY0nZzBxGI6KrV.svg", alt="Configuracões.", width="24", height="24" %} [**Configurações** > **Ignorar lista**]( /docs/devtools/settings/ignore-list/). Você também pode ocultar códigos fontes ignorados da árvore de arquivos com {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/N5Lkpdwpaz4YqRGFr2Ks.svg", alt="Menu de três pontos.", width="24", height="24" %} > [**Ocultar arquivos fontes listados como ignorados**](/docs/devtools/javascript/reference/#hide-ignore-listed) {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/XfSWf04g2cwpnFcmp40m.svg", alt="Experimental.", largura="20", altura="20" %}.
```
[the remark this answers is not in the document]
Comment (alvarocamillont) [reply site/pt/blog/new-in-devtools-112/index.md]: Done!

**Review by thiagowfx — APPROVED** [review]

File: site/pt/_partials/devtools/whats-new.md [hunk 2]
```diff
@@ -4,23 +4,23 @@ Consulte a versão em inglês do <a href="/tags/new-in-devtools/" translate="no"
...
-* [Marking logpoints and conditional breakpoints in the Console](/pt/blog/new-in-devtools-112/#logpoint)
-* [Ignore irrelevant scripts during debugging](/pt/blog/new-in-devtools-112/#ignore-list)
-* [JavaScript Profiler deprecation started](/pt/blog/new-in-devtools-112/#js-profiler-deprecation)
-* [Emulate reduced contrast](/pt/blog/new-in-devtools-112/#reduced-contrast)
+### Chrome 112 {: #chrome112 }
+
+* [Atualizações do gravador](/pt/blog/new-in-devtools-112/#recorder)
+* [Suporte para extensões de replay](/pt/blog/new-in-devtools-112/#replay-extensions)
+* [Gravação com seletores pierce](/pt/blog/new-in-devtools-112/#pierce-selectors)
+* [Exportar como um script Puppeteer com análise Lighthouse](/pt/blog/new-in-devtools-112/#puppeteer-lighthouse)
```
Comment (andreban) [inline site/pt/_partials/devtools/whats-new.md:12]:
```suggestion
* [Exportar gravações como um script Puppeteer com análise Lighthouse](/pt/blog/new-in-devtools-112/#puppeteer-lighthouse)
```

File: site/pt/_partials/devtools/whats-new.md [hunk 3]
```diff
@@ -4,23 +4,23 @@ Consulte a versão em inglês do <a href="/tags/new-in-devtools/" translate="no"
...
-* [Elements > Styles updates](/pt/blog/new-in-devtools-112/#elements-styles)
-* [CSS documentation in the Styles pane](/pt/blog/new-in-devtools-112/#css)
-* [CSS nesting support](/pt/blog/new-in-devtools-112/#nesting)
-* [Marking logpoints and conditional breakpoints in the Console](/pt/blog/new-in-devtools-112/#logpoint)
-* [Ignore irrelevant scripts during debugging](/pt/blog/new-in-devtools-112/#ignore-list)
-* [JavaScript Profiler deprecation started](/pt/blog/new-in-devtools-112/#js-profiler-deprecation)
-* [Emulate reduced contrast](/pt/blog/new-in-devtools-112/#reduced-contrast)
+### Chrome 112 {: #chrome112 }
+
+* [Atualizações do gravador](/pt/blog/new-in-devtools-112/#recorder)
```
Comment (andreban) [inline site/pt/_partials/devtools/whats-new.md:9]: I think "Recorder" / "Gravador" is a proper noun.
```suggestion
* [Atualizações do Gravador](/pt/blog/new-in-devtools-112/#recorder)
```

File: site/pt/_partials/devtools/whats-new.md [hunk 4]
```diff
@@ -4,23 +4,23 @@ Consulte a versão em inglês do <a href="/tags/new-in-devtools/" translate="no"
...
+* [Suporte para extensões de replay](/pt/blog/new-in-devtools-112/#replay-extensions)
```
Comment (andreban) [inline site/pt/_partials/devtools/whats-new.md:10]: Clarifying this is a feature added to the Recorder.

```suggestion
* [Extensões de replay no Gravador](/pt/blog/new-in-devtools-112/#replay-extensions)
```

File: site/pt/blog/new-in-devtools-112/index.md [hunk 5]
```diff
@@ -26,202 +25,234 @@ draft: true
   5. Update the sites/pt/_partials/devtools/whats-new.md file -->
 
 
-<!-- ## Recorder updates {: #recorder } -->
-
+<!-- ## Recorder updates {: #recorder }  -->
+## Atualizações do gravador {: #recorder }
```
Comment (andreban) [inline site/pt/blog/new-in-devtools-112/index.md:29]:
```suggestion
## Atualizações do Gravador {: #recorder } 
```

File: site/pt/blog/new-in-devtools-112/index.md [hunk 6]
```diff
@@ -26,202 +25,234 @@ draft: true
...
+<!-- ## Recorder updates {: #recorder }  -->
+## Atualizações do gravador {: #recorder } 
 <!-- ### Replay extensions support {: #replay-extensions } -->
-
+### Suporte para extensões de replay {: #replay-extensions }
 <!-- The **Recorder** introduces support for custom replay options that you can embed into DevTools with an extension. -->
-
+O **Gravador** apresenta o suporte para opções de replay personalizadas que você pode incorporar no DevTools com uma extensão.
 <!-- Try out the [example extension](https://github.com/puppeteer/replay/tree/main/examples/chrome-extension-replay). Select the new custom replay option to open the custom replay UI. -->
+Experimente a [extensão de exemplo](https://github.com/puppeteer/replay/tree/main/examples/chrome-extension-replay). Selecione a nova opção de reprodução personalizada para abrir a UI de replay personalizada.
```
Comment (andreban) [inline site/pt/blog/new-in-devtools-112/index.md:35]:
```suggestion
Experimente a [extensão de exemplo](https://github.com/puppeteer/replay/tree/main/examples/chrome-extension-replay). Selecione a nova opção de reprodução personalizada para abrir a interface de replay personalizada.
```

File: site/pt/blog/new-in-devtools-112/index.md [hunk 7]
```diff
@@ -26,202 +25,234 @@ draft: true
...
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/CAQFVtHyds7ByB0YMZht.png", alt="Custom replay UI.", width="800", height="563" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/CAQFVtHyds7ByB0YMZht.png", alt="UI de replay personalizada.", width="800", height="563" %}
```
Comment (andreban) [inline site/pt/blog/new-in-devtools-112/index.md:37]:
```suggestion
{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/CAQFVtHyds7ByB0YMZht.png", alt="Interface de replay personalizada.", width="800", height="563" %}
```

**Review by andreban — APPROVED** [review]
Dropped a few notes, but overall LGTM.

**Review by khaosdoctor — COMMENTED** [review]
I guess all typos were taken care of, LGTM

**Comment by alvarocamillont** [comment]
All Done! Thanks @andreban @thiagowfx  e @khaosdoctor

**Comment by jecfish** [comment]
Thanks all! Ready to go @AaronForinton

**Review by sofiayem — APPROVED** [review]

**Pull request merged**